code/modules/projectiles/guns/energy/projectile/sst.dm

```diff
@@ -1,136 +1,136 @@
-	icon = 'icons/obj/guns/energy/humility.dmi'
-	wielded_item_state = "_doble"
-	icon_state = "humility"
-	item_state = "humility"
-	charge_cost = 100
-	fire_sound = 'sound/weapons/energy/energy_shotgun.ogg'
-	gun_tags = list(GUN_PROJECTILE, GUN_LASER, GUN_ENERGY)
-	w_class = ITEM_SIZE_HUGE
-	force = WEAPON_FORCE_PAINFUL
-	charge_meter = FALSE
-	twohanded = TRUE
-	suitable_cell = /obj/item/cell/medium
-	slot_flags = SLOT_BACK
-	matter = list(MATERIAL_PLASTEEL = 30, MATERIAL_STEEL = 20, MATERIAL_SILVER = 15, MATERIAL_GOLD = 12, MATERIAL_PLATINUM = 0.5)
-	price_tag = 2500
-	init_recoil = CARBINE_RECOIL(1.3)
-	projectile_type=/obj/item/projectile/bullet/shotgun/beanbag/soporific
-	can_dual = FALSE
-	damage_multiplier = 1.2
-	fire_delay = 18
-
-/obj/item/gun/energy/sst/humility/update_icon()
-	..()
-	var/iconstring = initial(icon_state)
-	var/itemstring = ""
-
-	if(wielded)
-		itemstring += "_doble"
-
-	icon_state = iconstring
-	set_item_state(itemstring)
-
-/obj/item/gun/energy/sst/humility/preloaded
-
-/obj/item/gun/energy/sst/humility/preloaded/New()
-	cell = new /obj/item/cell/medium/moebius/high(src)
-	. = ..()
-	update_icon()
-
-/obj/item/gun/energy/sst/systemcost
-	name = "\"SST System Cost\" light machinegun"
-	desc = "\"Soteria Surface Tension\" brand light machinegun. The true answer to a horde of either monsters or rubberneckers in the medical bay. \
-	Able to fire soporific coated bullets at a high rate of fire, the System Cost is a trigger-happy trauma medic's dream come true."
-	icon = 'icons/obj/guns/energy/system_cost.dmi'
-	icon_state = "system_cost"
-	item_state = "system_cost"
-	fire_sound = 'sound/weapons/guns/fire/system_cost.ogg'  // Sky Driver sound from Eris
-	wielded_item_state = "_doble"
-	projectile_type=/obj/item/projectile/bullet/rifle_75/rubber/soporific
-	can_dual = FALSE
-	price_tag = 3200
-	charge_meter = FALSE
-	slot_flags = SLOT_BACK
-	charge_cost = 300
-	suitable_cell = /obj/item/cell/large
-	init_recoil = CARBINE_RECOIL(1.2)
-	can_dual = FALSE
-	twohanded = TRUE
-	gun_tags = list(GUN_PROJECTILE, GUN_LASER, GUN_ENERGY)
-	w_class = ITEM_SIZE_HUGE
-	force = WEAPON_FORCE_PAINFUL
-	matter = list(MATERIAL_PLASTEEL = 35, MATERIAL_STEEL = 25, MATERIAL_SILVER = 15, MATERIAL_GOLD = 12, MATERIAL_PLATINUM = 0.5)
-	damage_multiplier = 1.2
-	init_firemodes = list(
-		SEMI_AUTO_NODELAY,
-		FULL_AUTO_300
-		)
-
-/obj/item/gun/energy/sst/systemcost/update_icon()
-	..()
-	var/iconstring = initial(icon_state)
-	var/itemstring = ""
-
-	if(wielded)
-		itemstring += "_doble"
-
-	icon_state = iconstring
-	set_item_state(itemstring)
+/obj/item/gun/energy/sst
+	name = "\"SST Abnegate\" handgun"
+	desc = "\"Soteria Surface Tension\" brand handgun. A cooperative project between Soteria Medical and Soteria Research, the Abnegate uses Greyson tech to internally synthesize soporific coated 9mm bullets. \
+	Initially designed at the request of paramedics for a proper way to sedate rowdy armored targets from a distance."

```

Comment:
Line's broken out of the text string. Make sure that there's no space between the \ and the next line, just type \ then hit return.